modules/account/src/Volo.Abp.Account.Application/Volo/Abp/Account/AccountAppService.cs

```diff
@@ -62,19 +62,16 @@ public virtual async Task SendPasswordResetCodeAsync(SendPasswordResetCodeDto in
 
         public virtual async Task ResetPasswordAsync(ResetPasswordDto input)
         {
-            using (CurrentTenant.Change(input.TenantId))
```

Comment:
I think we still continue to use TenantId here, otherwise there is no way to switch to the host side.

Reply:
https://github.com/abpframework/abp/pull/9510/commits/81f1f7cb260a84576c9a439832eb95d17bcd0d35